modules/virt-querying-metrics.adoc

```diff
@@ -31,6 +31,14 @@ A value above '0' means that the vCPU wants to run, but the host scheduler canno
 ====
 To query the vCPU metric, the `schedstats=enable` kernel argument must first be applied to the `MachineConfig` object. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler.
 ====
+The `kubevirt_vmi_vcpu_delay_seconds_total` metric reports the cumulative time that a vCPU was enqueued by the host scheduler but could not run immediately. This delay is exposed to the virtual machine (VM) as steal time and can affect performance, often indicating CPU overcommitment or contention on the host.
+
+.Example vCPU delay query
```

Comment:
This example should actually go after the `.Example for the vCPU wait time query` to match the order that the queries are included in the section.

Check out line 91 for a good example of a metrics section that includes more than one type of metric and more than one query example for how to structure this,.